scenarios/kubernetes_build.py

```diff
@@ -71,6 +78,20 @@ def main(args):
     else:
         check('make', 'release')
     check('../release/push-build.sh', *push_build_args)
+    # log push-build location to path child jobs can find
+    # (gs://<shared-bucket>/$PULL_REFS/bazel-build-location.txt)
+    # first get build location
+    bucket = args.release or "kubernetes-release-dev"
+    dest = "ci"
+    if args.suffix:
+        dest += args.suffix
+    build_tar = tarfile.open("_output/release-tars/kubernetes.tar.gz")
+    latest = build_tar.getmember("kubernetes/version").read().strip()
+    gcs_build = "gs://"+bucket+dest+latest
+    # and then write it to GCS
+    gcs_shared = args.gcs_shared + os.getenv('PULL_REFS', '') + '/build-location.txt'
+    upload_string(gcs_shared, gcs_build)
```

Comment:
I'd make the extra steps optional, since only pr build job need to upload to shared builds for now.

Maybe consider only do this when `PULL_REFS` is valid.